### DISCUSSION TITLE
yiwen-blog-website/blog/charles

### DISCUSSION THREAD

**Discussion by giscus** [Hao-yiwen/yiwen-blog-website#266] [Announcements]
# yiwen-blog-website/blog/charles

今天在开发android，在开发过程中在思考一个问题，如果是从0开始，并没有完善的基建，那么如何知晓接口返回的情况那，于是打算重新熟悉charles进行抓包

https://hao-yiwen.github.io/yiwen-blog-website/blog/charles

**Comment by Hao-yiwen** [comment]
proxyman也是一个好工具。
